### PR TITLE
Documentation: add readme file for eventlog-server

### PR DESCRIPTION
**Pull request by Ruoyu-y**
* Add readme file for the eventlog-server service

### CODE REVIEW

File: service/eventlog-server/README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+To further verify the integrity and authenticity of the measurements in the confidential cloud native environment and its underlying platform, event logs are absolutely needed.
+By reviewing the event logs, user can identify any errors or issues that may be preventing the confidential environment from functioning correctly.
+
+This service (which is actually a GRPC server) will provide support to fetch the event logs for confidential cloud native environments, including TDX RTMR event logs and TPM event logs.
+
+
+
+## Introduction
+
+This service provides functionality to fetch event logs for confidential cloud native environments, both platform level(`PAAS` option) and container level (`SAAS` option). Using this service, user can fetch event logs for both TDX RTMR (`TDX_EVENTLOG` option) and TPM (`TPM_EVENTLOG` option).
```
Comment (hwang37) [inline service/eventlog-server/README.md:12]: Use "PaaS", "SaaS"

File: service/eventlog-server/README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+
+## Installation
+
+The Eventlog server service can be deployed as either DaemonSet or sidecar in different user scenarios.
+
+
+### Prerequisite
+
+To simplify the deployment process, we use Helm as the basic tool for deployment. Please install Helm following the [Helm official guide](https://helm.sh/docs/intro/install/).
```
Comment (hwang37) [inline service/eventlog-server/README.md:53]: by following

File: service/eventlog-server/README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+### Build docker image
+
+The dockerfile for the service can be found under `container/eventlog-server` directory. Use the following command to build the image:
+
+```
+cd ../..
+docker build -t ccnp_eventlog_server:0.1 -f container/eventlog-server/Dockerfile .
+```
+> Note: if you are using containerd as the default runtime for kubernetes. Please remember to use the following command to import the image into containerd:
```
Comment (hwang37) [inline service/eventlog-server/README.md:63]: if you are using containerd as the default runtime for kubernetes, please don't forget to use the following commands to import the image into containerd first:

File: service/eventlog-server/README.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+### User application deployment
+
+Make sure that the user application mount the same event log directory into the container, which defined by `eventlogDir` variable within the helm chart values.yaml.
+So that it can get the event log fetched and use according to their usage.
+
+
+
+## Testing
+
+One can play with service on host from the source code following the steps below:
```
Comment (hwang37) [inline service/eventlog-server/README.md:95]: User can play with service on host by following the steps below:

File: service/eventlog-server/README.md (hunk 1 above)
Comment (Ruoyu-y) [reply service/eventlog-server/README.md:12]: Same thing as the measurement one.

**Review by kenplusplus — APPROVED** [review]
LGTM

**Pull request merged**